package.json

```diff
@@ -32,7 +32,7 @@
   "scripts": {
     "predocs": "npm run compile",
     "docs": "jsdoc -c .jsdoc.js",
-    "system-test": "mocha build/system-test --timeout 600000 --exit",
+    "system-test": "mocha build/system-test --timeout 600000 --bail --exit",
```

Comment:
Not sure I like having either `--bail` or `--exit` here.  Was the intent to keep that in?

Reply:
I added bail here just while I was testing, but it's something we used to have. Since the system test runs take so long, it's convenient to have them exit early if we knew they were going to fail anyway. Do you know why `--bail` was removed?

Reply:
Just a ping-o on this one. I don't feel too strongly either way, but what I said above is a nice perk. I just can't remember if there was a good reason to not bail early.

Reply:
It's only really a problem if multiple system tests are failing, and you feel like you're whacking down one at a time.  That having been said, these things are crazy prone to cascading failures so I see where you're coming from.  I have no real problem with `--bail`, I just like consistency across the board.

I still think `--exit` is code smell 👅